app/adapters/alma_adapter.rb

```diff
@@ -11,28 +13,26 @@ def initialize(connection: AlmaAdapter::Connector.connection)
   # @param id [String] e.g. id = "991227830000541"
   # @see https://developers.exlibrisgroup.com/console/?url=/wp-content/uploads/alma/openapi/bibs.json#/Catalog/get%2Falmaws%2Fv1%2Fbibs Values that could be passed to the alma API
   # get one bib record is supported in the bibdata UI and in the bibliographic_controller
-  # @return [MARC::Record]
-  def get_bib_record(id, suppressed: true)
-    get_bib_records([id], suppressed:)&.first
+  # @return [AlmaAdapter::MarcRecord]
+  def get_bib_record(id, show_suppressed: false)
```

Comment:
I found the `suppressed` argument (which displayed suppressed records when false, and did not display them when true) confusing.  I switched the behavior around and tried to clarify the name.